### ISSUE TITLE
Add a sqitch remove command

### ISSUE THREAD

**Issue by rdunklau** [theory/sqitch#53]
It would be useful to have a remove command.

This command would take a change name, remove it from the plan and remove the associated deploy, revert and test script.

This should only work when no other change has a dependency on the named change.

**Comment by chamini2** [comment]
I was thinking of the remove command like a `git revert`, it should run the `revert` for the specified migration and add a line in the plan like

```
tables/products/create # Creating the products table
tables/tags/create # Creating the tags table
tables/product_is_tagged/create [tables/tags/create tables/products/create] # Creating an intermediate table to link products to tags
@1.0.0 2017-01-10 ... # First release
```

Then deciding the `tags` table will be of no use after all:

```
$ sqitch remove tables/tags/create
The following migrations would be removed:
tables/product_is_tagged/create
tables/tags/create
Remove migrations? [Yes]
```

And this would only change the `.plan` file, making it look like the following:

```
tables/products/create # Creating the products table
tables/tags/create # Creating the tags table
tables/product_is_tagged/create [tables/tags/create tables/products/create] # Creating an intermediate table to link products to tags
@1.0.0 2017-01-10 ... # First release
REMOVE tables/product_is_tagged/create
REMOVE tables/tags/create
```

And running the deploy is what really drops the tables  (running the reverts):

```
$ sqitch deploy
REMOVE tables/product_is_tagged/create ..... ok
REMOVE tables/tags/create .................. ok
```

**Comment by theory** [comment]
You might be interested in #21, @chamini2, where I mention that there is already syntax for adding revert changes to the plan file, but I haven't through through the ramifications of how it ought to work. I would consider that separate from this issue, though, where I think the idea is just to remove a change from the plan file.

**Comment by gpwatsona** [comment]
Yes to this, I just added a change and forgot the --require option and can't find a way of adding the tag while keeping the plan nice and tidy, unless I've missed something

**Comment by theory** [comment]
You can just add the required item to the new line in the plan file, @gpwatsona. Or, if you're using source control, revert the change (`git checkout sqitch.plan`) and add it again.
